working_with_TheDressler.py

```diff
@@ -7,6 +7,6 @@
                     const=sum, default=max,
                     help='sum the integers(default: find the max)')
 
-args =parser.parse_args()
+args= parser.parse_args()

```

Comment:
Before and after :-), like:  `args = parser.parse_args()`. See also: "[Whitespace in Expressions and Statements](https://pep8.org/#whitespace-in-expressions-and-statements)".